shared/src/components/timetable/timetableCard/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useMemo } from 'react';
+import { Timetable } from '../../../network/models/psu-tools/timetable';
+import classStatus from './classStatus';
+
+interface Props {
+  classData: Timetable.Class;
+  classDate: Timetable.Day;
+}
+
+const TimetableCard = ({ classData, classDate }: Props) => {
+  const time = useMemo(() => {
+    const date = new Date(classDate.date);
+    let timeStr = date.toLocaleTimeString('ru', {
+      hour: 'numeric',
+      minute: 'numeric',
+    });
+    timeStr = timeStr.replace(/^0+(\d)/, '$1');
+    return timeStr;
+  }, [classDate]);
+
+  const isClassInProgress = classStatus(classDate.date);
+
+  const cardClassName = `${isClassInProgress ? 'bg-c_accent dark:bg-c_accent text-c_bg-block shadow-[0.06rem_0.06rem_0.3rem_0_#D01F3680] dark:shadow-[0.06rem_0.06rem_0.3rem_0_#D01F3680]' : 'bg-cd_main shadow-[0_0_0.3rem_0_#EDEDED] dark:shadow-none dark:bg-cd_bg-block'}  w-screen rounded-[2.5rem]`;
```

Comment:
Для dark в обоих случаях нужно убрать тень
И желательно не использовать "магические" цвета, лучше использовать готовые из палитры либо на крайний случай создать еще один в палитре.
```suggestion
  const cardClassName = `${isClassInProgress ? 'bg-c_accent dark:bg-c_accent text-c_bg-block shadow-[0.06rem_0.06rem_0.3rem_0_var(--c\_shadow)]' : 'bg-cd_main shadow-[0_0_0.3rem_0_#EDEDED] dark:bg-cd_bg-block'} dark:shadow-none w-screen rounded-[2.5rem]`;
```